module/API/Controller.cpp

```diff
@@ -19,50 +24,71 @@ int Controller::limitPwmValue(const int value)
 }
 
 // 右モータにPWM値をセット
-void Controller::setRightMotorPwm(const int pwm)
+void Controller::setRightMotorPwm(const double pwm)
 {
+  pwmOfRightWheel = pwm;
   rightWheel.setPWM(limitPwmValue(pwm));
 }
 
 // 左モータにPWM値をセット
-void Controller::setLeftMotorPwm(const int pwm)
+void Controller::setLeftMotorPwm(const double pwm)
 {
+  pwmOfLeftWheel = pwm;
   leftWheel.setPWM(limitPwmValue(pwm));
 }
 
 // 右モータのPWM値をリセット
 void Controller::resetRightMotorPwm()
 {
+  pwmOfRightWheel = 0.0;
   rightWheel.reset();
 }
 
 // 左モータのPWM値をリセット
 void Controller::resetLeftMotorPwm()
 {
+  pwmOfLeftWheel = 0.0;
   leftWheel.reset();
 }
 
 // タイヤのモータを停止する
 void Controller::stopMotor()
 {
+  pwmOfRightWheel = 0.0;
+  pwmOfLeftWheel = 0.0;
   leftWheel.stop();
   rightWheel.stop();
 }
 
 // アームのモータにPWM値をセット
-void Controller::setArmMotorPwm(const int pwm)
+void Controller::setArmMotorPwm(const double pwm)
 {
+  pwmOfArm = pwm;
   armMotor.setPWM(limitPwmValue(pwm));
 }
 
 // アームのモータのPWM値をリセット
 void Controller::resetArmMotorPwm()
 {
+  pwmOfArm = 0;
```

Comment:
```suggestion
  pwmOfArm = 0.0;
```
上のを見た感じだと、double 型の変数をセットしてるみたいだから、一応 0.0 の方がいいかな。